src/ldp/http/SparqlUpdateBodyParser.ts

```diff
@@ -25,12 +24,9 @@ export class SparqlUpdateBodyParser extends BodyParser {
   }
 
   public async handle({ request, metadata }: BodyParserArgs): Promise<SparqlUpdatePatch> {
-    const options = { objectMode: request.readableObjectMode };
-    const toAlgebraStream = pipeSafely(request, new PassThrough(options));
-    const dataCopy = pipeSafely(request, new PassThrough(options));
```

Comment:
Are we still doing this anywhere else?

Reply:
From what I could see this was the only place where this still happened, which is why it didn't really surprise my there was a problem here.